android-plugin/src/test/files/social/privacy/GoogleTrackerCheck.java

```diff
@@ -0,0 +1,4 @@
+import com.google.android.gms.analytics.Tracker; // Noncompliant {{Using com.google.android.gms.analytics.* is a potential threat for privacy.}}
```

Comment:
We should test more imports cases : 

- import com.google.android.gms.analytics.*
- import com.google.android.gms.Test
- import com.google.android.gms.analytics.UneAutreClasse
- import com.io.ecocode.MaClasse